Makefile.am

```diff
@@ -617,6 +618,14 @@ builtin_modules += android_wakelock
 builtin_sources += plugins/android-wakelock.c
 endif
 
+if ACCOUNTSSETTINGS
```

Comment:
Ok, good catch. Still, I do not like to add "ubuntu" prefixes around to things that are imho pretty general. I can easily see us adding watches for system properties in the accounts-settings plugins that come from general interfaces different to "com.ubuntu.touch.AccountsService.Phone" (like the general properties that AccountsService has for users).

Note also that in case "com.ubuntu.touch.AccountsService.Phone" is not defined in the system, there is no misbehaviour: properties are not tracked and default values are used.